src/AutoRest.CSharp/Common/Output/Models/Types/SchemaObjectType.cs

```diff
@@ -223,6 +224,51 @@ protected override ObjectTypeConstructor BuildSerializationConstructor()
                 serializationInitializers.Add(new ObjectPropertyInitializer(RawDataField, deserializationParameter, null));
             }
 
+            if (ModelTypeMapping != null)
+            {
+                foreach (var propertyWithSerialization in ModelTypeMapping.GetPropertiesWithSerialization())
+                {
+                    if (serializationInitializers.Exists(i => i.Property.Declaration.Name == propertyWithSerialization.Name))
+                        continue;
+                    var csharpType = BuilderHelpers.GetTypeFromExisting(propertyWithSerialization, typeof(object), MgmtContext.TypeFactory);
+                    var isReadOnly = propertyWithSerialization switch
+                    {
+                        IPropertySymbol propertySymbol => propertySymbol.SetMethod == null,
+                        IFieldSymbol fieldSymbol => fieldSymbol.IsReadOnly,
+                        _ => throw new NotSupportedException($"'{propertyWithSerialization.ContainingType.Name}.{propertyWithSerialization.Name}' must be either field or property.")
+                    };
+                    var xml = propertyWithSerialization.GetDocumentationCommentXml();
+                    string comment = "";
+                    if (!string.IsNullOrEmpty(xml))
+                    {
+                        var xd = new global::System.Xml.XmlDocument();
+                        xd.LoadXml(xml);
+                        comment = xd.SelectSingleNode("/member/summary")?.InnerText.Trim() ?? "";
+                    }
```

Comment:
this is redundant - we are adding this property because this property has been defined in the customized code, therefore no matter how well we did for this property which will be written into the generated code, it will be removed again in post processor, because this property has been defined in the customization code and it cannot be defined in two places.
Please refer how it is done in `ModelTypeProviderFields`: https://github.com/RodgeFu/autorest.csharp/blob/seralize-add-property/src/AutoRest.CSharp/Common/Output/Models/Types/ModelTypeProviderFields.cs#L305
Here we just need to put a placeholder for summary, description, etc.

Reply:
seems the comment here will impact the parameter's comment in the generated code like below. please help to confirm.
![image](https://github.com/Azure/autorest.csharp/assets/6038235/bef46375-2501-404d-bccf-816ebd29dada)

Reply:
looks like we did not consider it.
Then we should have this, and we should change it as well in `ModelTypeProviderFields`.
Please write a method in a common place to get the description from a symbol and use it here and in `ModelTypeProviderFields`

Reply:
set the description to string.empty to keep things simple consider all these methods are internal